src/main/java/com/redhat/pantheon/model/QuerySingleResult.java

```diff
@@ -0,0 +1,20 @@
+package com.redhat.pantheon.model;
+
+import java.util.Map;
+
+/**
+ * A simple model to represent a single element result when running queries.
+ */
+public class QuerySingleResult {
```

Comment:
I'm not sure we need this. If the only thing this is keeping is the result itself (and no additional information), why not just return the result directly. We are only adding one additional level to the json response. i.e:
```json
{
   "result": 
       "module" {
       }
}
```
vs
```json
"module" {
}
```
that intermediate `result` section is adding no value.

Reply:
Well, I wanted to maintain consistency between APIs we are building, we can remove it and just return the module object WDUT? @carlosmunoz

Reply:
I don't think we need API's to be as consistent for the single-return ones. I mean, if the `result` object is going to potentially have other properties I say we leave it, but if we are just keeping it to maintain the same nomenclature at the first level, then there's no real reason to keep it. 

Based on this I say we remove it, and if we need to change it later (because we add more data to the response for example), then we handle it.